test/e2e/common/container_probe.go

```diff
@@ -263,7 +263,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 							},
 							InitialDelaySeconds: 15,
 							TimeoutSeconds:      5,
-							FailureThreshold:    1,
+							FailureThreshold:    5, // to accommodate nodes which are slow in bringing up containers.
```

Comment:
does this start to get close to the observation timeout? if the container was restarted at the end of 5 failures, would that happen within the observation timeout and fail the test as we would want?

Reply:
`defaultObservationTimeout` is 2m.  default probe `periodSeconds` is 10s.  So `InitialDelaySeconds + periodSeconds * FailureThreshold = 65s.   So that should be fine.

Reply:
ok